### ISSUE TITLE
Work In Progress badge

### ISSUE THREAD

**Issue by dy** [badges/stability-badges#4]
Hi @hughsk!

I always use these labels, but one badge I oftentimes find needed but absent here is _work in progress_ badge. Factually that is a status of project, indicating that it is too early to use it.

What are thoughts about that?

Thanks.

**Comment by BLUUUUUE** [comment]
Hello @dfcreative you could create own badge at [![h](https://img.shields.io/badge/Press%20this%20link-shields.io-red.svg?link=http//shields.io

**Comment by BLUUUUUE** [comment]
don't press that because will not work here

**Comment by dy** [comment]
Yeah, now I do like that. I hoped it may become a more recognizable status badge, because factually it is.

**Comment by Potherca** [comment]
This might merely be a difference in definitions, but why not use "Experimental" or "Unstable" for "work-in-progress"? 

To me they would both indicate "too early to use"...

**Comment by dy** [comment]
@Potherca the difference is that _WIP_ oftentimes is not ready to be used and going to be finished, like under construction, whereas _experimental_ is proof of concept, i. e. finished to some extent.
For example [font-atlas](https://github.com/hughsk/font-atlas) is finished but experimental, whereas [audio](https://github.com/audiojs/audio/tree/class) is work in progress - not ready to be used at all, but planning to be somewhat stable.

**Comment by Potherca** [comment]
@dfcreative Thanks for elaborating! I can see the difference between "WIP" and "Experimental".
I still have trouble grokking the difference between "WIP" and  "Unstable".

To quote Robert Burns:

> The best laid schemes o’ Mice an’ Men Gang aft agley

In other words, regardless whether a project _might_ at some point become stable, while work is underway it _is not_ stable. 

To my mind, WIP = Unstable (Although not all Unstable project are necessarily WIP). It might just boil down to my perspective that "Unstable" expresses the intent to (someday) become "Stable".

(I hope you don't mind my attempt to better understand the difference. Project life-cycle/stages is a topic that genuinely fascinate me. Enough to have made [my own attempt at a definition](https://github.com/potherca-blog/project-stages/)).
